sycl/test-e2e/USM/fill_any_size.cpp

```diff
@@ -0,0 +1,91 @@
+// RUN: %{build} -o %t1.out
+// RUN: %{run} %t1.out
+// UNSUPPORTED: (opencl && cpu)
+// clang-format off
+// UNSUPPORTED-TRACKER: https://github.com/oneapi-src/unified-runtime/issues/2440
+// clang-format on
```

Comment:
```suggestion
// clang-format off
// UNSUPPORTED: (opencl && cpu)
// UNSUPPORTED-TRACKER: https://github.com/oneapi-src/unified-runtime/issues/2440
// clang-format on
```
IIUC, there should be no additional line between UNSUPPORTED and UNSUPPORTED-TRACKER.

Reply:
That's why we had the CI failure: https://github.com/intel/llvm/actions/runs/12657653976/job/35273434387?pr=16544#step:17:322

Reply:
Thanks, your explanation and fix look correct to me